src/main/java/com/epam/test_generator/controllers/test_result/TestResultTransformer.java

```diff
@@ -0,0 +1,26 @@
+package com.epam.test_generator.controllers.test_result;
+
+import com.epam.test_generator.controllers.test_result.response.TestResultDTO;
+import com.epam.test_generator.entities.TestResult;
+
+public class TestResultTransformer {
+    public static TestResultDTO toDto(TestResult testResult) {
```

Comment:
Why you made it static?
Other transformers are spring components, which are injected in spring context.